Automattic/wp-calypso
# Help Contact Form: move Redux state.contactForm.isShowingQandASuggestions to local state

Removes Redux state slice that's only used locally by one component and moves it to local state. The `resetContactForm` dispatch in `InlineHelpPopover` is not needed as the state automatically resets on mount/remount.

I'm also removing several `INLINE_HELP_SEARCH_REQUEST` action types that are not used since help search was converted to React Query in #55753.

**How to test:**
In the Inline Help popover, enter some searchable term to the chat box, wait a second until background `/me/qanda` search finishes and the button changes to "Continue", and the click "Continue". That toggles the `showingQandAStep` state and a screen with search results is shown. See the video:

![Screen Capture on 2021-09-17 at 08-37-05](https://user-images.githubusercontent.com/664258/133739654-dbf31c60-56c7-4237-8cf6-095403e99b50.gif)
